format/nexus.py

```diff
@@ -1632,8 +1640,8 @@ def __getitem__(self, index):
         d = self.lookup[index]
         i = index - self.offset[d]
         N, height, width = self.datasets[d].shape
-        data_as_flex = dataset_as_flex_int(
-            self.datasets[d].id.id,
```

Comment:
Is it correct that `.id.id` did nothing? I would find it surprising if @jmp1985 left this in there for no reason. I assume that the code worked, though...